### DISCUSSION TITLE
New constellation-lines

### DISCUSSION THREAD

**Discussion by seescho** [Stellarium/stellarium#1615] [Q&A] [answered]
I had always problems to recognize the constellations in the night-sky. It´s 6 years ago, I decided to take photos of the sky and paint my own constellation-lines into these photos. Some of my constellations look like the historical ones, other are more or less  similar to those of H.A.Ray. If you are interested in, you can have a look on my homepage: https://sternenkarten.com/

Since more than a year, my constellation-lines are part of kstars. In kstars they are called: Western (sternenkarten.com)

Kstars, indi and stellarsolver are great pieces of software for my abservatory. But for preparing my observations, I prefer stellarium. And my 3 years old grandson love stellarium as well. Now I want to show him my constellation-lines in stellarium.

So I started transferring my constellationlines from kstars into my local installation of stellarium yesterday. It looks like that´s a lot of work. For painting the constellation-lines in kstars, the star-numbers in the HD-catalog are used. In stellarium, I have to use the HIP-numbers.

To make this work easier: Does anybody of you know a tool to transfer the HD-numbers into HIP-numbers? If not, I will do this by hand, star by star. But this will take it´s time. Yesterday I just managed transferring 4 out of 88 constellations :)

Edgar

**Comment by gzotti** [comment, marked as answer]
Converting HD to HIP is not part of Stellarium. You will need about 22 cloudy evenings for your private skyculture, or find/develop a tool elsewhere.

**Reply by seescho** [reply]
Thanks for your answer. I thought, that´s not that easy.
Yesterday I converted looking at the stars in stellarium. Just now I tried the Simbad-database. This work much better for my needs.  Now I need only 15 cloudy nights for converting, maybe. :)
Edgar

**Reply by alex-w** [reply]
Perl/Python script + SIMBAD/XMatch ;)

**Reply by seescho** [reply]
ThankYou Alex
Unfortunatly I am not familiar with Scripting in Perl or Python. So I did the transformation manually. My skyculture is now working on my local copy of stellarium and I´m happy with this :)
Doing this work, I found some constellation-lines to enhance. I´m working on this atm.
